pkg/tests/schema_pulumi_test.go

```diff
@@ -1508,3 +1509,263 @@ runtime: yaml
 		})
 	}
 }
+
+func TestConfigureGetRaw(t *testing.T) {
+	getOkExists := func(d *schema.ResourceData, key string) (interface{}, bool) {
+		v := d.GetRawConfig().GetAttr(key)
+		if v.IsNull() {
+			return nil, false
+		}
+		return d.Get(key), true
+	}
+	resMap := map[string]*schema.Resource{
+		"prov_test": {
+			Schema: map[string]*schema.Schema{
+				"test": {
+					Type:     schema.TypeString,
+					Optional: true,
+				},
+			},
+		},
+	}
+
+	runConfigureTest := func(t *testing.T, configPresent bool) {
+		tfp := &schema.Provider{
+			ResourcesMap: resMap,
+			Schema: map[string]*schema.Schema{
+				"config": {
+					Type:     schema.TypeString,
+					Optional: true,
+				},
+			},
+			ConfigureContextFunc: func(ctx context.Context, rd *schema.ResourceData) (interface{}, diag.Diagnostics) {
+				_, ok := getOkExists(rd, "config")
+				require.Equal(t, configPresent, ok, "Unexpected config value")
+				return nil, nil
```

Comment:
```
ConfigureContextFunc is the function used to configure a Provider.

The interface{} value returned by this function is stored and passed into the subsequent resources as the meta parameter. This return value is usually used to pass along a configured API client, a configuration structure, etc.
```

What would make this test stronger is getting the result of `d.GetRawConfig().GetAttr("config")` and returning it, and then accessing it again in the resource through the meta parameter.

Reply:
If that's too involved and arguably testing something unrelated, we can at least assert on the value of `d.GetRawConfig().GetAttr("config")` to make sure it matches expectation.